mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg

```diff
@@ -26,6 +26,15 @@ If tradeable = false, then even after you've discovered the item it still won't
 IF THE WaitForDiscovery OPTION ISN'T SHOWING UP, BE SURE TO RUN THE GAME AT LEAST ONCE WITH BetterTrader INSTALLED! THE OPTION WILL BE GENERATED ON STARTUP!
 
 #Better Trader#
+Amber: Stack = 1 Price = 5 SellPrice = 5 Tradeable = true IgnoreWaitForDiscovery = true Sellable = true
+AmberPearl: Stack = 1 Price = 10 SellPrice = 10 Tradeable = true IgnoreWaitForDiscovery = true Sellable = true
+Ruby: Stack = 1 Price = 20 SellPrice = 20 Tradeable = true IgnoreWaitForDiscovery = true Sellable = true
+HelmetYule: Stack = 1 Price = 100 SellPrice = 100 Tradeable = true IgnoreWaitForDiscovery = true Sellable = true
+HelmetDverger: Stack = 1 Price = 620 SellPrice = 620 Tradeable = true IgnoreWaitForDiscovery = true Sellable = true
+BeltStrength: Stack = 1 Price = 950 SellPrice = 950 Tradeable = true IgnoreWaitForDiscovery = true Sellable = true
+YmirRemains: Stack = 1 Price = 120 SellPrice = 120 Tradeable = true IgnoreWaitForDiscovery = true Sellable = true
```

Comment:
I think the stock item prices also need to go up (especially considering the precious commodities price boosts)

Reply:
Only Ymir Remains went up for now.  Will monitor